tests/ensemble/test_ranger_forest_classifier.py

```diff
@@ -114,6 +114,23 @@ def test_importance(self, iris_X, iris_y, importance, scale_permutation_importan
             else:
                 assert rfc.importance_mode_ == 3
 
+    def test_importance_pvalues(self, digits_X, digits_y, importance):
+        rfc = RangerForestClassifier(importance=importance)
+
+        if importance not in ["none", "impurity", "impurity_corrected", "permutation"]:
+            with pytest.raises(ValueError):
+                rfc.fit(digits_X, digits_y)
+            return
+
+        if not importance == "impurity_corrected":
+            with pytest.raises(Exception):
+                rfc.fit(digits_X, digits_y)
+                rfc.get_importance_pvalues()
```

Comment:
to be explicit about where this exception is raised
```suggestion
            rfc.fit(digits_X, digits_y)
            with pytest.raises(ValueError):
                rfc.get_importance_pvalues()
```